build.py

```diff
@@ -23,4 +28,98 @@ def decrypt():
 @task(_checkdir)
 def encrypt():
     '''Encrypt secrets.dec.tf file.'''
-    os.system("ansible-vault encrypt --output secrets.enc secrets.dec.tf && rm -rf secrets.dec.tf")
\ No newline at end of file
+    os.system("ansible-vault encrypt --output secrets.enc secrets.dec.tf && rm -rf secrets.dec.tf")
+
+@task(_checkdir)
+def layer_dependency(summary='False',quiet='False'):
+    """Check layer dependency from the current layers"""
+    try:
+        remote_states = _layer_dependency(path.get_working_path())
+    except Exception as e:
+        print('Error getting deps:',e)
+
+    if summary.lower() == 'true':
+        deps = {'this': []}
+        for rs in remote_states:
+            deps['this'].append(remote_states[rs]['key'])
+        remote_states = deps
+
+    try:
+        if quiet == 'False': print("Note layer dependency is calculated using remote states.\nNevertheless, other sort of dependencies could exist without this kind of resources,\ne.g. if you relay on some resource created in a different layer and not referenced here.")
+        if len(remote_states) > 1:
+          if len(remote_states) > 2:
```

Comment:
2 space indentation? :crying_cat_face:

Reply:
yeah, well... my IDE decided to do that... fixed... 😄